doc/reference/trd104-syscalls.md

```diff
@@ -423,22 +437,22 @@ start at address 0x0. However, even if they do begin at 0x0, the
 Tock Binary Format for application images mean that the first address
 will not be executable code and so 0x0 will not be a valid function.
 In the case that 0x0 is valid application code and where the
-linker places a callback function, the first instruction of the function
+linker places an upcall function, the first instruction of the function
 should be a no-op and the address of the second instruction passed
 instead.
 
 If a userspace process invokes subscribe on a driver ID that is not
 installed in the kernel, the kernel MUST return a failure with an
-error code of NOSUPPORT and a callback of the Null Callback.
+error code of NOSUPPORT and an upcall of the Null Upcall.
```

Comment:
```suggestion
error code of `NODEVICE` and an upcall of the Null Upcall.
```

Per line 272? Although per the code it's `NOSUPPORT`.

Reply:
Good catch.